lib/stacks/eks-blueprint-stack.ts

```diff
@@ -61,12 +61,21 @@ export class EksBlueprint extends cdk.Stack {
 
         const clusterInfo = clusterProvider.createCluster(this, vpc, blueprintProps.version ?? KubernetesVersion.V1_19);
 
+        const postDeploymentSteps = Array<ClusterPostDeploy>();
+
         for (let addOn of (blueprintProps.addOns ?? [])) { // must iterate in the strict order
             addOn.deploy(clusterInfo);
+            const postDeploy : any  = addOn;
+            if((postDeploy as ClusterPostDeploy).postDeploy !== undefined) {
+                postDeploymentSteps.push(<ClusterPostDeploy>postDeploy);
+            }
         }
+        
         if (blueprintProps.teams != null) {
             blueprintProps.teams.forEach(team => team.setup(clusterInfo));
         }
+
+        postDeploymentSteps.forEach(step => step.postDeploy(clusterInfo, blueprintProps.teams ?? []));
```

Comment:
step is really an `addon` yea? Consider changing name to `postDeploymentAddons`? Something like that so we are being explicit?

Reply:
It is an addon at the moment, it just so happens that some addons can also contain post deployment steps. Once we get a use case for pure bootstrapper, we can expose this collection through properties.